pointcloud_crop_box/config/pointcloud_crop_box_params.yaml

```
@@ -1,13 +1,14 @@
 /**:
   pointcloud_crop_box:
     ros__parameters:
-      input_topic: "/panther/velodyne_points"
-      output_topic: "/points_filtered"
-      target_frame: "panther/base_footprint"
+      target_frame: panther/base_footprint
       min_x: -0.405
       max_x: 0.405
       min_y: -0.424
       max_y: 0.424
       min_z: -0.05
       max_z: 0.5
       negative: true
+      message_type: laserscan #or pointcloud
+      input_topic: /panther/rplidar/scan
+      output_topic: "/scan_filtered"
```

I see default output is pointcloud, so name `scan_filtered` can be misleading
```suggestion
      output_topic: "/points_filtered"
```